.github/workflows/actions.yml

```diff
@@ -45,42 +47,44 @@ jobs:
         with:
           arguments: --refresh-dependencies --stacktrace --scan clean build
       - name: Sonarqube
-        if: matrix.java == '8' && matrix.os == 'ubuntu-latest'
+        if: matrix.java == '8' && matrix.os == 'ubuntu'
         env:
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
           SONAR_TOKEN: "61ab2579215aa8a0024a2f9368fc1298fdecfd18"
         run: ./gradlew jacocoTestReport sonarqube --stacktrace -i
 
-  # This will be handled on a later stage
-  # deploy:
-  #   runs-on: ubuntu-latest
-  #   name: Deploy
-  #   if: github.ref == 'refs/heads/master'
-  #   steps:
-  #     - name: Check out repo
-  #       uses: actions/checkout@v2
-  #     - name: Setup java
-  #       uses: actions/setup-java@v1
-  #       with:
-  #         java-version: 8
-  #     - name: Cache Gradle
-  #       uses: actions/cache@v1
-  #       with:
-  #         path: ~/.gradle/caches
-  #         key: ${{ runner.os }}-gradle-${{ hashFiles('**/*.gradle*') }}
-  #         restore-keys: ${{ runner.os }}-gradle-
-  #     - name: Perform Release
-  #       uses: eskatos/gradle-command-action@v1
-  #       if:
-  #       with:
-  #         arguments: ciPerformRelease
-  #     - name: Upload Release to GitHub
-  #       uses: softprops/action-gh-release@v1
-  #       if: startsWith(github.ref, 'refs/tags/')
-  #       with:
-  #         files: ./build/libs/junit-pioneer-*.jar
-  #       env:
-  #         GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+  deploy:
+    needs: build
+    runs-on: ubuntu-latest
+    name: Deploy with Shipkit
+    if: github.ref == 'refs/heads/master'
+    steps:
+      - name: Sanitize Branch
+        uses: frabert/replace-string-action@master
+        id: gitbranch
+        with:
+          pattern: 'refs/heads/(.*)'
+          string: "${{ github.ref }}"
+          replace-with: '$1'
+      - uses: actions/checkout@v2
+      - name: Setup java
+        uses: actions/setup-java@v1
+        with:
+          java-version: 8
+      - name: Perform Release
+        uses: eskatos/gradle-command-action@v1
+        env:
+          TRAVIS_BRANCH: ${{ steps.gitbranch.outputs.replaced }}
+          TRAVIS_COMMIT_MESSAGE: ${{ github.event.head_commit.message }}
+          TRAVIS_PULL_REQUEST: ${{ github.event_name == 'pull_request' }}
```

Comment:
What do we need these environment variables for?

Reply:
that is the funny part, shipkit is only compatible with travis currently, and needs those travis environment variables. So i "fake" to be travis to get shipkit working, till they hopefully accept my proposal, and i can finish https://github.com/mockito/shipkit/pull/868